### PR TITLE
Fix several errors

### PR DESCRIPTION
**Pull request by mariabernis**
#### :tophat: What? Why?

We had several unresolved warnings on the repo and if we want to use it as a framework (`use_frameworks!` in Podfile) we need to fix those. 
Changed project settings to mark warnings as errors.

### CODE REVIEW

File: PNChart/PNBarChartGrouped.h [hunk 1]
```diff
@@ -10,11 +10,6 @@
 #import "PNChartDelegate.h"
 #import "PNBar.h"
 
-#define xLabelMargin 15
-#define yLabelMargin 15
```
Comment (saulbaro) [inline PNChart/PNBarChartGrouped.h:14]: @mariabernis where did the margins go? Can't find them. Is that an error or they weren't needed?

File: PNChart/PNBarChart.h [hunk 2]
```diff
@@ -10,10 +10,6 @@
 #import "PNChartDelegate.h"
 #import "PNBar.h"
 
-#define xLabelMargin 15
-#define yLabelMargin 15
```
Comment (saulbaro) [inline PNChart/PNBarChart.h:14]: @mariabernis where did the margins go? Can't find them. Is that an error or they weren't needed?

Comment (fernandogcat) [inline PNChart/PNBarChart.h:14]: I think you should ask kevinzhow instead of @mariabernis

**Comment by fegabe** [comment]
✋ are you merging to master, is that ok?

File: PNChart/PNBarChart.h (hunk 2 above)
Comment (fernandogcat) [inline PNChart/PNBarChart.h:14]: Ok, I thought we're just updating the library!

**Comment by fernandogcat** [comment]
👍

File: PNChart/PNBarChartGrouped.h (hunk 1 above)
Comment (mariabernis) [inline PNChart/PNBarChartGrouped.h:14]: They were not used.

**Comment by mariabernis** [comment]
@fegabe We only have master, yes.

File: PNChart/PNBarChart.h (hunk 2 above)
Comment (mariabernis) [inline PNChart/PNBarChart.h:14]: This is a fork with different features than the original repo. Updating it to latest version is not straight forward, I'm just fixing our errors so the pod compiles in universal.

**Comment by saulbaro** [comment]
💬 👍

**Pull request merged**